lib/crypto/ed25519/ed25519_test.go

```diff
@@ -99,3 +99,19 @@ func TestNewKeypairFromMnenomic_Again(t *testing.T) {
 	expectedPubkey := common.MustHexToBytes("0xf56d9231e7b7badd3f1e10ad15ef8aa08b70839723d0a2d10d7329f0ea2b8c61")
 	require.Equal(t, expectedPubkey, kp.Public().Encode())
 }
+
+func TestVerifySignature(t *testing.T) {
```

Comment:
Can you have a test with the verification failing as well please?

You could use test cases and subtests, [example](https://github.com/ChainSafe/gossamer/blob/e7fbfe5ef8241ad0d50850c8b3d7ae417f428f35/lib/trie/hash_test.go#L834-L874)